packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -655,7 +655,7 @@ abstract class StatelessWidget extends Widget {
 ///
 /// This is a skeleton of a stateful widget subclass called `YellowBird`.
 ///
-/// In this example. the [State] has no actual state. State is normally
+/// In this example the [State] has no actual state. State is normally
```

Comment:
Nit: this is the proper punctuation. :)

```suggestion
/// In this example, the [State] has no actual state. State is normally
```